src/app/utils/item.js

```diff
@@ -211,10 +212,11 @@ function LibraryItem() {
     const mappedItemSource = itemSourceMappings[itemSource];
     const isOffsite = this.isOffsite(holdingLocation.prefLabel.toLowerCase());
     let url = null;
-    const isSerial = bib && bib.issuance && bib.issuance[0]['@id'] === 'urn:biblevel:s';
+    const isSerial = !!(bib && bib.issuance && bib.issuance[0]['@id'] === 'urn:biblevel:s');
```

Comment:
why are we changing this?

Reply:
This was causing tests to fail, because it was generating `0` instead of `false` for some reason.